test/e2e/bootstrapper_test.go

```diff
@@ -161,3 +178,38 @@ func isKubeletRunning(t *testing.T, logPath string) bool {
 	assert.NoError(t, err)
 	return strings.Contains(string(buf), "Started kubelet")
 }
+
+// getSvcPath returns the fully qualified path of the specified service.
+func getSvcPath(svcName string) (string, error) {
+	svcPath := ""
+	svcMgr, err := mgr.Connect()
+	if err != nil {
+		return svcPath, fmt.Errorf("could not connect to Windows SCM: %s", err)
+	}
+	defer svcMgr.Disconnect()
+	mySvc, err := svcMgr.OpenService(svcName)
+	if err != nil {
+		// Could not find the service, so it was never created
+		return svcPath, err
+	}
+	defer mySvc.Close()
+	config, err := mySvc.Config()
+	if err != nil {
+		return svcPath, err
+	}
+	return config.BinaryPathName, nil
+}
+
+// populateKubeletArgs populates the map kubeletargs with kubelet service arguments that have path
+func populateKubeletArgs(path string) {
```

Comment:
You can also collect pass in the args that you need and just collect the ones you are going to look at.